### ISSUE TITLE
Wrong number of probes in Slack status notification 

### ISSUE THREAD

**Issue by nicnocquee** [hyperjumptech/monika#512]
**Describe the bug**
The number of probes in status notification sent to slack is not correct. 

**Screenshots**

![64B14EE7-E453-4EBE-AA20-84C24D2600AA](https://user-images.githubusercontent.com/311343/139234676-84463581-2c50-4a77-91e6-7d534dcc6b02.jpeg)

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Version 1.6.5

**Comment by mishijima** [comment]
Hello @nicnocquee 

I believe the issue is because the `numberOfProbes` comes from the result of `getProbesSummarySql`, see https://github.com/hyperjumptech/monika/blob/main/src/components/logger/history.ts#L432

Based on my findings, I think the `numberOfProbes` should be set with the size of `config.probes` array.

<img width="615" alt="Screen Shot 2021-11-02 at 8 20 52 pm" src="https://user-images.githubusercontent.com/5510786/139820129-0fdf61e9-cd14-43bb-8054-a3e15a9ef3fa.png">

**My config**
```
notifications:
  - id: unique-id-slack
    type: slack
    data:
      url: <slack-webhook>
probes:
  - id: sample_1
    name: Sample Request
    requests:
      - method: GET
        url: http://localhost:3000
    alerts:
      - query: response.time > 200
        message: Response time is {{ response.time }} ms, expecting less than 200ms
    incidentThreshold: 2
    recoveryThreshold: 2
  - id: sample_2
    name: Sample Request
    requests:
      - method: GET
        url: http://localhost:3000
    alerts:
      - query: response.status != 200
        message: Status code is not 200. I think their service is down.
    incidentThreshold: 2
    recoveryThreshold: 2
```

**Comment by nicnocquee** [comment]
> Hello @nicnocquee
> 
> I believe the issue is because the `numberOfProbes` comes from the result of `getProbesSummarySql`, see https://github.com/hyperjumptech/monika/blob/main/src/components/logger/history.ts#L432
> 
> Based on my findings, I think the `numberOfProbes` should be set with the size of `config.probes` array.
> 
> <img alt="Screen Shot 2021-11-02 at 8 20 52 pm" width="615" src="https://user-images.githubusercontent.com/5510786/139820129-0fdf61e9-cd14-43bb-8054-a3e15a9ef3fa.png">
> 
> **My config**
> 
> ```
> notifications:
>   - id: unique-id-slack
>     type: slack
>     data:
>       url: <slack-webhook>
> probes:
>   - id: sample_1
>     name: Sample Request
>     requests:
>       - method: GET
>         url: http://localhost:3000
>     alerts:
>       - query: response.time > 200
>         message: Response time is {{ response.time }} ms, expecting less than 200ms
>     incidentThreshold: 2
>     recoveryThreshold: 2
>   - id: sample_2
>     name: Sample Request
>     requests:
>       - method: GET
>         url: http://localhost:3000
>     alerts:
>       - query: response.status != 200
>         message: Status code is not 200. I think their service is down.
>     incidentThreshold: 2
>     recoveryThreshold: 2
> ```

Thanks for the hint! I made a PR to fix this. #524

**Comment by nicnocquee** [comment]
Fix in #524
